chapters/03/exercise.md

```diff
@@ -94,20 +94,11 @@ select * from renamed
 ```
 with source as (
 
-  select * from {{ source('dbt_seeds', 'ga_traffic') }}
-
-), renamed as (
-
-  select
-    date,
-    sessions,
-    users as visitors,
```

Comment:
We have certainly applied renaming here so why do we want to remove it now?

Reply:
I have renamed also the columns in .csv to the correct one, the main idea was to avoid unnecessary steps during the workshop and make it more smooth, but if you insist we can leave it there.

Reply:
That make sense, thanks for explaining :)